framework/global/internal/interactive.cpp

```diff
@@ -147,16 +147,16 @@ void Interactive::message(Type type, const std::string& title, const std::string
     msgBox.exec();
 }
 
-mu::io::path Interactive::selectOpeningFile(const QString& title, const QString& dir, const QString& filter)
+mu::io::path Interactive::selectOpeningFile(const QString& title, const io::path& dir, const QString& filter)
 {
-    QString path = QFileDialog::getOpenFileName(mainWindow()->qMainWindow(), title, dir, filter);
-    return io::pathFromQString(path);
+    QString path = QFileDialog::getOpenFileName(mainWindow()->qMainWindow(), title, dir.toQString(), filter);
```

Comment:
Maybe return a result directly?

Reply:
I think it doesn't matter, it's easier to debug or log, and we don't need any performance here, and the compiler probably optimizes.